.travis-ghcjs.sh

```diff
@@ -0,0 +1,12 @@
+#/usr/bin/env bash
+set -ev
+
+# tinc
+#cabal exec which hspec-discover
+#barf
+cabal install hspec-discover --prefix $HOME/.local
```

Comment:
instead of having a separate file, we could just have one `.travis.sh` file and branch on the `GHCJS` env variable.